src/components/shared/MarkerTooltipContents.js

```diff
@@ -308,41 +334,82 @@ function getMarkerDetails(
                       formatBytes
                     )
                   )}
-                  {nursery.cur_capacity === undefined
-                    ? null
-                    : _markerDetail(
+                  {nursery.cells_tenured && nursery.cells_allocated_nursery
+                    ? _markerDetail(
+                        'gcpromotioncells',
+                        'Cells tenured',
+                        formatValueTotal(
+                          nursery.cells_tenured,
+                          nursery.cells_tenured +
+                            nursery.cells_allocated_nursery,
+                          formatSI
+                        )
+                      )
+                    : null}
+                  {nursery.cur_capacity
+                    ? _markerDetail(
                         'gcnurseryusage',
                         'Bytes used',
                         formatValueTotal(
                           nursery.bytes_used,
                           nursery.cur_capacity,
                           formatBytes
                         )
-                      )}
-                  {nursery.new_capacity === undefined
-                    ? null
-                    : _markerDetail(
+                      )
+                    : null}
+                  {nursery.new_capacity
+                    ? _markerDetail(
                         'gcnewnurserysize',
                         'New nursery size',
                         nursery.new_capacity,
                         formatBytes
-                      )}
-                  {nursery.lazy_capacity === undefined
-                    ? null
-                    : _markerDetail(
+                      )
+                    : null}
+                  {nursery.lazy_capacity
+                    ? _markerDetail(
                         'gclazynurserysize',
                         'Lazy-allocated size',
                         nursery.lazy_capacity,
                         formatBytes
-                      )}
-                  {nursery.chunk_alloc_us === undefined
-                    ? null
-                    : _markerDetail(
+                      )
+                    : null}
+                  {nursery.cells_allocated_nursery &&
+                  nursery.cells_allocated_tenured
+                    ? _markerDetail(
+                        'gcnursaryallocations',
+                        'Nursery allocations since last minor GC',
+                        formatValueTotal(
+                          nursery.cells_allocated_nursery,
+                          nursery.cells_allocated_nursery +
+                            nursery.cells_allocated_tenured,
+                          formatSI
+                        )
+                      )
+                    : null}
+                  {evictTime
+                    ? _markerDetail(
+                        'gctenurerate',
+                        'Tenure rate (bytes/sec)',
```

Comment:
do we really need the bytes/sec in the label, if it's in the unit already ?

Reply:
I'm trying to avoid a problem (and maybe you have a suggestion) where "Tenure rate" means two different things, and this is the first use of the new meaning.  I don't want the other GC developers to assume the first meaning when they use the profiler.

 * First meaning: The percentage of cells promoted from the nursery when it is collected.
 * Second meaning: number of cells per seconjd the GC can promote from the nursery.

So I put the units here in the label to hint at the 2nd meaning, in case they wern't obvious in the number.

@hotsphink, @jonco3 what do you think?

Reply:
What about 'survival rate' for the proportion of objects that are tenured and 'tenured allocation rate' for the second just to be really clear?  I agree that we probably shouldn't call them the same thing.